### PR TITLE
Book-keeping before release

### PR DESCRIPTION
**Pull request by atreyasha**
### Checklist

- [x] Search issues/pull-requests to ensure no duplicates
- [x] Update cram unit tests: irrelevant
- [x] Update `zsh`, `fish` and `bash` shell-completions: irrelevant
- [x] Update mandoc: irrelevant
- [x] Update usage function and readme: irrelevant
- [x] Update locales and add new translations

### Description

Just some book-keeping stuff before release:

1. Updated the changelog assuming a minor bump, but I can change that if we agree to something else.

2. Modified the version filtering example in the readme to have single quotes instead of double, since the double ones might lead to undesired variable expansions.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
-## [Ongoing](https://github.com/pbrisbin/downgrade/compare/v8.1.1...master)
+## [Ongoing](https://github.com/pbrisbin/downgrade/compare/v8.2.0...master)
 
 None
 
+## [v8.2.0](https://github.com/pbrisbin/downgrade/compare/v8.1.1...v8.2.0)
+
+- [NEW] Pacman cache directory accumulation to support multiple cache locations
+- [NEW] Removal of --nosudo CLI option, downgrade now expects root privileges
```
Comment (pbrisbin) [inline CHANGELOG.md:8]: This feels like a big change, the entire interface is completely changed. I was thinking of `v9` if only to signal that. On the other hand, misuse is not harmful, the error is clear, and the behavior once executed has not changed in a meaningful way... So I'm fine with only a minor bump.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,12 @@
-## [Ongoing](https://github.com/pbrisbin/downgrade/compare/v8.1.1...master)
+## [Ongoing](https://github.com/pbrisbin/downgrade/compare/v8.2.0...master)
 
 None
 
+## [v8.2.0](https://github.com/pbrisbin/downgrade/compare/v8.1.1...v8.2.0)
+
+- [NEW] Pacman cache directory accumulation to support multiple cache locations
```
Comment (pbrisbin) [inline CHANGELOG.md:7]: Just noting here that when I read "cache directory accumulation" I keep thinking it means the cache is accumulating files, not that the options accumulate. Not sure if it's worth rewording in the CHANGELOG?

> - [NEW] Now accepts multiple cache directory options

?

**Review by pbrisbin — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (atreyasha) [reply CHANGELOG.md:8]: Hmm true, it is indeed a big change and a major bump would definitely indicate that. I had qualms about consecutive major bumps taking place in the last few releases but I guess the changes are what they are and the versioning should reflect that. Major bump it is :)

File: CHANGELOG.md (hunk 2 above)
Comment (atreyasha) [reply CHANGELOG.md:7]: Aha yes. Will change.

**Comment by atreyasha** [comment]
v9.0.0 released :birthday:

File: CHANGELOG.md (hunk 1 above)
Comment (pbrisbin) [reply CHANGELOG.md:8]: FWIW, I treat Arch packages very differently than almost any other versioning situation. In a library, doing a major version update probably results in a large set of users that will refuse to move up and a maintenance headache if you've committed to supporting older versions. In Arch, everyone will just `-Syu` and it'll be fine. I still try to apply the same rules about indicating kinds of changes, but there's zero hesitation about large jumps in version.

**Pull request merged**